apple_result.csv

```diff
@@ -0,0 +1,3 @@
+supply,188
+buy,115
+result,73
```

Comment:
this and following the result files shouldn't be pushed to the remove repo

Reply:
+